troubleshooting/debugging.md

```diff
@@ -608,6 +609,23 @@ In other cases, the original config would be correct. For example, assume your a
 
 ### Fluent Bit Windows containers
 
+#### Enabling debug mode for Fluent Bit Windows images
+Debug mode can be enabled for Fluent Bit Windows images so that when Fluent Bit crashes, we are able to generate crash dumps which would help in debugging the issue. This can be accomplished by overriding the entrypoint command to include the `-EnableCoreDump` flag. For example-
+```
+# If you are using the config file located at C:\fluent-bit\etc\fluent-bit.conf 
+Powershell -Command C:\\entrypoint.ps1 -EnableCoreDump
+
+# If you are using a custom config file located at other location say C:\fluent-bit-config\fluent-bit.conf. 
+Powershell -Command C:\\entrypoint.ps1 -EnableCoreDump -ConfigFile C:\\fluent-bit-config\\fluent-bit.conf
```

Comment:
just curious why double backslash?

Reply:
It is just escaping the backslashes. These may not be necessary but since Windows treats double backslashes as singles ones, I put them there.